components/EditEventForm.js

```diff
@@ -9,6 +9,10 @@ import InputField from './InputField';
 import StyledButton from './StyledButton';
 import TimezonePicker from './TimezonePicker';
 
+const defaultStartsAt = new Date();
+defaultStartsAt.setHours(19);
+defaultStartsAt.setMinutes(0);
```

Comment:
I'm not a fan of having `defaultStartsAt` as a global variable, could we wrap that in a `getDefaultDate` helper?